### PR TITLE
[Block Storage]: data migration tutorial 

### PR DESCRIPTION
**Pull request by nerda-codes**

### CODE REVIEW

File: storage/block/api-cli/managing-a-volume.mdx [hunk 1]
```diff
@@ -213,4 +213,3 @@ The file is now transferred. You can connect to your Instance again, and use the
```
Comment (bene2k1) [inline storage/block/api-cli/managing-a-volume.mdx:215]:
```suggestion
```
no need for so many empty lines at the end :)

File: tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx [hunk 2]
```diff
@@ -0,0 +1,339 @@
+---
+meta:
+  title: Migrating Block Storage data to Block Storage Low Latency
+  description: This page shows you how to migrate Block Storage data to Block Storage Low Latency
```
Comment (bene2k1) [inline tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx:4]:
```suggestion
  description: This page shows you how to migrate Block Storage data to Block Storage Low Latency volume
```

File: tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx [hunk 3]
```diff
@@ -0,0 +1,339 @@
...
+content:
+  h1: Migrating Block Storage data to Block Storage Low Latency
+  paragraph: This page shows you how to migrate Block Storage data to Block Storage Low Latency
```
Comment (bene2k1) [inline tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx:7]:
```suggestion
  paragraph: This page shows you how to migrate Block Storage data to Block Storage Low Latency volume
```

File: tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx [hunk 4]
```diff
@@ -0,0 +1,339 @@
...
+  - instances
+  - block-storage
+tags: block-storage data migration block-storage-low-latency
+dates:
+  validation: 2024-02-15
+  posted: 2024-02-15
+---
+
+
+This tutorial shows you how to migrate your Instance's data from a Block Storage volume (b_ssd) to a Block Storage Low Latency according to your needs and level of expertise. You will find out how to:
```
Comment (bene2k1) [inline tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx:18]:
```suggestion
This tutorial shows you how to migrate your Instance's data from an existing Block Storage volume (b_ssd) to a Block Storage Low Latency volume. You will find out how to:
```

File: tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx [hunk 5]
```diff
@@ -0,0 +1,339 @@
...
+
+- prepare for migration
+- identify your volumes
+- identify which migration procedure to use, according to how your volumes are mounted
+- migrate your data without Logical Volume Management or Partition Table
+- migrate your data using **Logical Volume Management (LVM)**
+- migrate your data with Partition Table but without Logical Volume Management
+
+Note that your Operating System and the volume on which you have stored it, **cannot be migrated to a Block Storage Low Latency volume**.
```
Comment (bene2k1) [inline tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx:27]:
```suggestion
Note that your operating system and the volume on which you have stored it, **cannot be migrated to a Block Storage Low Latency volume**.
```

File: tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx [hunk 6]
```diff
@@ -0,0 +1,339 @@
...
+
+### Identifying which migration procedure to use
+
+1. Run the following command to check if your volumes are managed by LVM:
+
+    ```
+    pvs
+    ```
+
+    If an output similar to the following displays, with your source volume under the `PV`section, we recommend that you migrate your data using the [Migrating data with Logical Volume Management (LVM) procedure](#migrating-data-with-logical-volume-management-(lvm)).
```
Comment (bene2k1) [inline tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx:151]:
```suggestion
    If an output similar to the following displays, with your source volume under the `PV` section, we recommend that you migrate your data using the [Migrating data with Logical Volume Management (LVM) procedure](#migrating-data-with-logical-volume-management-(lvm)).
```

File: tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx [hunk 7]
```diff
@@ -0,0 +1,339 @@
...
+
+
+In the output, we can observe that:
+
+- the root system `/` is mounted using `/dev/sda1`, as `/dev/sda` is the System volume. As stated previously in this documentation, data contained in this volume must not be migrated.
+- `/dev/sdc` is mounted as a non-partitioned raw device, since there is no number appended to the device path. For this use case, you must migrate your data using the [Migrating data without Partition Table or LVM procedure](#migrating-data-without-partition-table-or-lvm).
+- `/dev/sde1` is a partitioned raw device. The naming convention used for it (sde1) is typical for identifying partitions on storage devices. For this use case, one must migrate their data using the [Migrating data without LVM but with Partition Table procedure](#migrating-data-without-lvm-but-with-partition-table).
+
+
+## Migrating data without Partition Table or LVM
```
Comment (bene2k1) [inline tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx:187]:
```suggestion
## Migrating data without partition table or LVM
```

File: tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx [hunk 8]
```diff
@@ -0,0 +1,339 @@
...
+2. Use the following command to make a copy of your Instance's `fstab` file:
+    ```
+    # This command creates an exact copy of the /etc/fstab file named fstab.backup
+
+    cp -a /etc/fstab /etc/fstab.backup
+    ```
+3. Stop any applications and/or services that may be accessing the data on the source volume.
+4. Unmount your Block SSD volume:
+    ```
+    umount /dev/sdb
```
Comment (nerda-codes) [inline tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx:310]:
```suggestion
    umount /dev/sdb1
```

File: tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx [hunk 9]
```diff
@@ -0,0 +1,339 @@
...
+       product: b_ssd
+       vendor: SCW
+       physical id: 0.2.0
+       bus info: scsi@2:0.2.0
+       logical name: /dev/sdd
+       version: v42
+       size: 23GiB (25GB)
+       capabilities: 5400rpm
+       configuration: ansiversion=5 logicalsectorsize=512 sectorsize=4096
+
```
Comment (nerda-codes) [inline tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx:113]: Need to add ``` here

**Review by bene2k1 — APPROVED** [review]

File: tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx [hunk 10]
```diff
@@ -0,0 +1,339 @@
...
+    lrwxrwxrwx 1 root root  9 Feb  7 16:25 /dev/disk/by-id/scsi-0SCW_b_ssd_volume-715131d8-60fe-4707-b199-71d0e005d76d -> ../../sdc
+    lrwxrwxrwx 1 root root  9 Feb  7 16:25 /dev/disk/by-id/scsi-0SCW_b_ssd_volume-7e1a9b98-b7ee-4bbc-8f8f-a7abbfffc937 -> ../../sda
+    lrwxrwxrwx 1 root root 10 Feb  7 16:25 /dev/disk/by-id/scsi-0SCW_b_ssd_volume-7e1a9b98-b7ee-4bbc-8f8f-a7abbfffc937-part1 -> ../../sda1
+    lrwxrwxrwx 1 root root 11 Feb  7 16:25 /dev/disk/by-id/scsi-0SCW_b_ssd_volume-7e1a9b98-b7ee-4bbc-8f8f-a7abbfffc937-part14 -> ../../sda14
+    lrwxrwxrwx 1 root root 11 Feb  7 16:25 /dev/disk/by-id/scsi-0SCW_b_ssd_volume-7e1a9b98-b7ee-4bbc-8f8f-a7abbfffc937-part15 -> ../../sda15
+    ```
+4. On the Scaleway console, click the <Icon name="more" /> icon next to your Instance's volume, then click **More info**. A pop-up displays with the volume's information.
+5. Check that for each volume, the ID displayed in the pop-up matches the one returned in the output in step 3. For example, the `/dev/sda` volume's ID on the console, matches the one returned in the output.
```
Comment (RoRoJ) [inline tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx:130]:
```suggestion
5. Check that for each volume, the ID displayed in the pop-up matches the one returned in the output in step 3. For example, the `/dev/sda` volume's ID on the console, should match the one returned in the output.
```

**Review by RoRoJ — APPROVED** [review]

File: tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx [hunk 11]
```diff
@@ -0,0 +1,339 @@
...
+- prepare for migration
+- identify your volumes
+- identify which migration procedure to use, according to how your volumes are mounted
+- migrate your data without Logical Volume Management or Partition Table
+- migrate your data using **Logical Volume Management (LVM)**
+- migrate your data with Partition Table but without Logical Volume Management
+
+Note that your operating system and the volume on which you have stored it, **cannot be migrated to a Block Storage Low Latency volume**.
+
+<Macro id="iam-requirements" />
```
Comment (jcirinosclwy) [inline tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx:29]:
```suggestion
<Macro id="requirements" />
```

File: tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx [hunk 12]
```diff
@@ -0,0 +1,339 @@
...
+
+<Message type="requirement">
```
Comment (jcirinosclwy) [inline tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx:31]:
```suggestion
```

File: tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx [hunk 13]
```diff
@@ -0,0 +1,339 @@
...
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
```
Comment (jcirinosclwy) [inline tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx:32]:
```suggestion
- A Scaleway account logged into the [console](https://console.scaleway.com)
```

File: tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx [hunk 14]
```diff
@@ -0,0 +1,339 @@
...
+  - You have [created an SSH key](/console/project/how-to/create-ssh-key/)
+  - You have [created an Instance](/compute/instances/how-to/create-an-instance/) and [attached a Block SSD volume](/storage/block/how-to/attach-a-volume/) to it, containing data to migrate
+  - Your Instance's OS is **not hosted on the Block SSD volume that contains the data you want to migrate**
+</Message>
```
Comment (jcirinosclwy) [inline tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx:36]:
```suggestion
```

File: tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx [hunk 15]
```diff
@@ -0,0 +1,339 @@
...
+        pvremove /dev/sdb
+        ```
+10. Restart your applications/services, and check that they are running.
+11. Reboot your Instance and check that your applications are running correctly.
+12. Optionally, wait a few days and [delete the SSD volume](/storage/block/how-to/delete-a-volume/) using the Scaleway console.
+
+## Migrating data without LVM but with Partition Table
+
+<Message type="important">
+  To follow this procedure you must have:
```
Comment (jcirinosclwy) [inline tutorials/migrate-block-storage-data-to-block-low-latency/index.mdx:289]:
```suggestion
  To follow this procedure, you must have:
```

**Review by jcirinosclwy — APPROVED** [review]

**Pull request closed without merging**